libnmstate/__init__.py

```diff
@@ -36,3 +41,12 @@
     "schema",
     "PrettyState",
 ]
+
+
+def get_version():
```

Comment:
I do not think we need to expose this, especially if we indent to use `__version__` instead.
@tyll , what do you think?

Reply:
Yes, you are right. Done!